rfc6962-bis.xml

```diff
@@ -1308,6 +1308,14 @@ but it is expected there will be a variety.
         <t>
           If the TLS client holds an STH that predates the SCT, it MAY, in the process of auditing, request a new STH from the log (<xref target="get-sth"/>), then verify it by requesting a consistency proof (<xref target="get-sth-consistency"/>). Note that if the TLS client uses <spanx style="verb">get-all-by-hash</spanx>, then it will already have the new STH.
         </t>
+        <section title="TLS Feature Extension">
+          <t>
+            If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required.
```

Comment:
I reworded it deliberately so that those questions didn't need to be asked.  ;-)

A TLS client evaluates whether or not "compliance" is required on a per-chain basis rather than a per-certificate basis, I think.  (A TLS server doesn't have to supply SCTs for every cert in the chain, so the notion of requiring each cert in a chain to individually be "compliant" makes no sense to me).

With my language, "compliance" is required when a TLS client encounters a chain that contains any cert(s) that have the transparency_info TLS extension indentifier in the TLS Feature extension.

We define "compliance" elsewhere.  Section 9.2, for example, says:
 "To be considered compliant, a certificate MUST be accompanied by at
   least one valid SCT or at least one valid inclusion proof."